### PR TITLE
Migrate Branding Colors

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Migrate Branding Colors component
Created and upgraded a wrapper directive for color picker

## Motivation and Context
Angular migration

## How Has This Been Tested?
Locally and [stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu I will update unit tests and remove the original directive file in the next PR. Please review. Thanks

File: src/scripts/components/color-picker/dtv-color-picker.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+angular.module('risevision.common.components')
+  .directive('colorPicker', [function () {
+      return {
+        restrict: 'E',
+        scope: {
+          ngModel: '=',
+          onBlur: '&?',
```
Comment (alex-deaconu) [inline src/scripts/components/color-picker/dtv-color-picker.js:9]: @ezequielc should we just use a different name for this to indicate a change, not necessarily triggered by blur? Thanks.

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good so far 👍

File: src/scripts/components/color-picker/dtv-color-picker.js (hunk 1 above)
Comment (ezequielc) [reply src/scripts/components/color-picker/dtv-color-picker.js:9]: @alex-deaconu Good point. I will update it to `ngChange` to be consistent with time picker. Thanks

**Pull request merged**